src/gui_common/SpeciesDetailsPanel.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using Godot;
+
+public class SpeciesDetailsPanel : PanelContainer
+{
+    [Export]
+    public NodePath SpeciesDetailsLabelPath = null!;
+
+    [Export]
+    public NodePath SpeciesPreviewPath = null!;
+
+    [Export]
+    public NodePath HexPreviewPath = null!;
+
+    [Export]
+    public NodePath FossilisationButtonPath = null!;
+
+    [Export]
+    public NodePath FossilisationDialogPath = null!;
+
+    private CustomRichTextLabel speciesDetailsLabel = null!;
+    private SpeciesPreview speciesPreview = null!;
+    private CellHexesPreview hexesPreview = null!;
+    private Button fossilisationButton = null!;
+    private FossilisationDialog fossilisationDialog = null!;
+
+    private Species? previewSpecies;
+
+    public Species? PreviewSpecies
+    {
+        get => previewSpecies;
+        set
+        {
+            previewSpecies = value;
+            UpdateSpeciesPreview();
+        }
+    }
+
+    public override void _Ready()
+    {
+        base._Ready();
+
+        speciesDetailsLabel = GetNode<CustomRichTextLabel>(SpeciesDetailsLabelPath);
+        speciesPreview = GetNode<SpeciesPreview>(SpeciesPreviewPath);
+        hexesPreview = GetNode<CellHexesPreview>(HexPreviewPath);
+        fossilisationButton = GetNode<Button>(FossilisationButtonPath);
+        fossilisationDialog = GetNode<FossilisationDialog>(FossilisationDialogPath);
+    }
+
+    /// <summary>
+    ///   Updates displayed species information based on the set preview species.
+    /// </summary>
+    private void UpdateSpeciesPreview()
+    {
+        speciesPreview.PreviewSpecies = PreviewSpecies;
+
+        if (PreviewSpecies is MicrobeSpecies microbeSpecies)
+        {
+            hexesPreview.PreviewSpecies = microbeSpecies;
+            fossilisationButton.Disabled = false;
+        }
+        else
+        {
+            fossilisationButton.Disabled = true;
+            GD.PrintErr("Unknown species type to preview: ", PreviewSpecies);
```

Comment:
This print will happen when the species is null. Is it intended to allow setting the species to preview to null?

Reply:
This follows the existing pattern of `SpeciesPreview`.